init.sh

```diff
@@ -3,3 +3,6 @@
 export FCCSWBASEDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source /cvmfs/fcc.cern.ch/sw/0.8.1/init_fcc_stack.sh $1
 
+# TODO: temporary, until new ACTS tag available & tests work with gcc6.2
+export CMAKE_PREFIX_PATH=$CMAKE_PREFIX_PATH:/cvmfs/fcc.cern.ch/sw/0.8pre/acts/0.3.0/x86_64-slc6-gcc49-opt/share/cmake/ACTS/
```

Comment:
This mixes acts that is compiled against LCG_83 with the rest of the stack that is compiled against LCG_88. The DD4hep installations are not compatible and this will lead to an inconsistent environment. Let me know if you'd like to have a `acts 0.3.0` install in 0.8.1 (for both gcc62 and gcc49)...